### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by Parikshit-Hooda**
# Issue #104 
<OVERALL SUMMARY OF PULL REQUEST>

### Changes
components -> component
added some necessary punctuations
changed 'contributors welcome' -> 'Contributions to this repository are welcome!'

### Flags
- <DESCRIBE ISSUES OR HOLDS FOR REVIEWERS>

### Related Issues
- Issue #<NUMBER HERE>
- Pull Request #<NUMBER HERE>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,22 +64,22 @@ In the project directory, you can run:
 
 #### `npm run dev`
 
-Runs the app in the development mode.<br>
-Open [http://localhost:3001/examples](http://localhost:3001/examples) to view it in the browser.
+   Runs the app in the development mode.<br>
+   Open [http://localhost:3001/examples](http://localhost:3001/examples) to view it in the browser.
```
Comment (irmerk) [inline README.md]: These edits can be removed.

File: README.md [hunk 2]
```diff
@@ -63,22 +64,22 @@ In the project directory, you can run:
...
 
 The page will reload if you make edits.<br>
 
 #### `npm run build`
 
-Builds the app for production to the `build` folder.<br>
-It correctly bundles React in production mode and optimizes the build for the best performance.
+   Builds the app for production to the `build` folder.<br>
+   It correctly bundles React in production mode and optimizes the build for the best performance.
```
Comment (irmerk) [inline README.md]: These edits can be removed.

File: README.md [hunk 3]
```diff
@@ -13,13 +13,14 @@ This repo contains two React-based editors:
...
-Using these editors you could allow people to either edit rich formatted text using
+Using these editors, you could allow people to either edit rich formatted text using
 markdown (and provide a WYSIWYG preview), or allow them to edit using a WYSIWYG
 editor and use markdown for persistence.
 
 The editor includes a formatting toolbar.
 
-This component is Apache-2 licensed Open Source. Contributors welcome!
+This component is Apache-2 licensed Open Source.
+Contributions to this repository are welcome!
```
Comment (irmerk) [inline README.md]: This edit can be removed.

**Comment by irmerk** [comment]
Also, you'll want to add a DCO (Developers Certificate of Originality) per [the instructions](https://github.com/accordproject/markdown-editor/blob/master/CONTRIBUTING.md):
> Ensure you provide a DCO sign-off for your commits using the -s option of git commit. For more information see [how this works](https://github.com/probot/dco#how-it-works).

File: README.md (hunk 1 above)
Comment (Parikshit-Hooda) [reply README.md]: ok

File: README.md (hunk 2 above)
Comment (Parikshit-Hooda) [reply README.md]: ok

**Comment by Parikshit-Hooda** [comment]
Let me close this PR and start afresh!

File: README.md (hunk 3 above)
Comment (Parikshit-Hooda) [reply README.md]: ok

**Pull request closed without merging**